spec.bs

```diff
@@ -168,12 +169,21 @@ are:
 1. If |contribution|["{{PAHistogramContribution/value}}"] is negative,
     [=exception/throw=] a {{RangeError}}.
 1. Let |scopingDetails| be [=this=]'s [=PrivateAggregation/scoping details=].
+1. Let |batchingScope| be the result of running |scopingDetails|' [=scoping
+    details/get batching scope steps=].
+1. Let |filteringIdByteSize| be 1.
+1. Let |preSpecifiedParamsMap| be the [=pre-specified report parameters map=].
+1. If |preSpecifiedParamsMap|[|batchingScope|] [=map/exists=]:
+    1. Set |filteringIdByteSize| to |preSpecifiedParamsMap|[|batchingScope|]'s
+        [=pre-specified report parameters/filtering ID byte size=].
+1. If |contribution|["{{PAHistogramContribution/filteringId}}"] is not in the
```

Comment:
`filteringId` is optional, shall we check whether it's null first?

Reply:
Ah good catch, done!